ssg/build_yaml.py

```diff
@@ -1154,6 +1154,13 @@ def make_refs_and_identifiers_product_specific(self, product):
         general_references = self.references.copy()
         for todel in product_references:
             general_references.pop(todel)
+        for ref in Rule.PRODUCT_REFERENCES:
```

Comment:
Does it make sense to move this up ahead of the pop? I caught the unqualified identifiers anyways with it where it was, but up to the team.